mobile_testkit_tests/test_generate_clusters_from_pool.py

```diff
@@ -88,7 +88,7 @@ def test_generate_clusters_from_pool_ip_to_node(cleanup_folder, cluster_conf, nu
     cluster_config_dir = test_data + "/cluster_configs/"
 
     # Run tests with mock_pool_ips.json for backward compatibility
-    generate_clusters_from_pool(ip_to_node_type_pool_file)
+    generate_clusters_from_pool(ip_to_node_type_pool_file, False)
```

Comment:
by default the tests are not testing docker so false will result in the same behavior as before.